apps/condo/domains/organization/schema/SendNewBillingReceiptFilesNotificationsService.js

```diff
@@ -0,0 +1,240 @@
+
+const fileAdapter = new FileAdapter(BILLING_RECEIPT_FILE_FOLDER_NAME)
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    ORGANIZATION_NOT_FOUND: {
+        mutation: 'sendNewBillingReceiptFilesNotifications',
+        code: BAD_USER_INPUT,
+        type: NOT_FOUND,
+        message: 'Could not find Organization by specified search criteria',
+    },
+    CREATED_AFTER_DATE_ILLEGAL_VALUE: {
+        mutation: 'sendNewBillingReceiptFilesNotifications',
+        code: BAD_USER_INPUT,
+        type: WRONG_VALUE,
+        message: 'Wrong value for createdAfter field. Required non null date string, less than current time',
+    },
+    INVALID_PERIOD_PROVIDED: {
+        mutation: 'sendNewBillingReceiptFilesNotifications',
+        variable: ['data', 'period'],
+        code: BAD_USER_INPUT,
+        type: NOT_FOUND,
+        message: `Please provide proper period not before start of previous and not later than start of current month, valid format is ${DATE_FORMAT}`,
+    },
+}
+
+function getWatermark (value) {
+    if (isEmpty(value)) {
+        return null
+    }
+
+    return dayjs(value)
+}
+
+async function prepareAttachments (files) {
+    return files.filter(file => !isEmpty(get(file, ['sensitiveDataFile', 'publicUrl'])))
+        .map(file => {
+            const {
+                sensitiveDataFile: {
+                    filename,
+                    publicUrl,
+                    mimetype,
+                    originalFilename,
+                },
+            } = file
+
+
+            const attachment = {
+                mimetype,
+                originalFilename,
+                publicUrl,
+            }
+
+            if (fileAdapter.acl && fileAdapter.acl.generateUrl) {
+                attachment.publicUrl = fileAdapter.acl.generateUrl({
+                    filename: `${fileAdapter.folder}/${filename}`,
+                    originalFilename,
+                })
+            }
+
+            return attachment
+        })
+}
+
+const SendNewBillingReceiptFilesNotificationsService = new GQLCustomSchema('SendNewBillingReceiptFilesNotificationsService', {
+    types: [
+        {
+            access: true,
+            type: 'input SendNewBillingReceiptFilesNotificationsInput { dv: Int!, sender: JSON!, organization: OrganizationWhereUniqueInput!, createdAfter: String!, period: String! }',
+        },
+        {
+            access: true,
+            type: 'type SendNewBillingReceiptFilesNotificationsOutput { notificationsSent: Int! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canSendNewBillingReceiptFilesNotifications,
+            schema: 'sendNewBillingReceiptFilesNotifications(data: SendNewBillingReceiptFilesNotificationsInput!): SendNewBillingReceiptFilesNotificationsOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
```

Comment:
It looks like a part of another task! Why it's not a part of existing task? We already have a task for sending notification for new receipts.

It's better to move this logic close to that task! It should be similar logic